src/backend/distributed/commands/table.c

```diff
@@ -221,14 +221,22 @@ PostprocessCreateTableStmtPartitionOf(CreateStmt *createStatement, const
 	if (IsCitusTable(parentRelationId))
```

Comment:
I think we could combine the new if with this one so it will be clearer that we distribute when:

- parentRelation is a Citus table
- relation is not a Citus table
- relation is a partition
- relation's parent is parentTable

Reply:
So you suggest a single `if` statement with 4 parts?

Reply:
Yes 👍